Makefile

```diff
@@ -29,7 +29,6 @@ ifndef NDK_VERSION
 NDK_VERSION=25.1.8937393
 endif
 # Set names of play cpp sdk library files
-MAC_FILE=play_cpp_sdk_Darwin_x86_64.tar.gz
 MACARM64_FILE=play_cpp_sdk_Darwin_arm64.tar.gz
```

Comment:
what about Mac intel. Not supported?

Reply:
it's supported with arm64, arm64 has another fat library which supports both of x86_64, arm64